### DISCUSSION TITLE
Typebox plugin

### DISCUSSION THREAD

**Discussion by crishoj** [formkit/formkit#1349] [Ideas]
I was looking for a [Typebox](https://github.com/sinclairzx81/typebox) equivalent of the [Zod validation plugin](https://formkit.com/plugins/zod), but couldn't immediately find one.

If I were to write a Typebox validation plugin myself, would it be advisable to base it off of the current Zod plugin design?

**Comment by fenilli** [comment]
Yes I would say so, there is a lot of things in the zod plugin that helps tie FormKit and it, so you can use those for the base.

**Reply by crishoj** [reply]
I took a stab at it: https://github.com/formkit/formkit/pull/1350

Your feedback is welcome.
